### PR TITLE
Update isOrganization util method

### PR DESCRIPTION
**Pull request by ThaminduR**
## Purpose

- Update `isOrganization` util method to handle the super tenant scenario.
- Refactor util method to `OrganizationManagementUtil` class.

### CODE REVIEW

**Review by AnuradhaSK — DISMISSED** [review]

File: components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 import org.wso2.carbon.identity.organization.management.service.exception.OrganizationManagementException;
 import org.wso2.carbon.identity.organization.management.service.exception.OrganizationManagementServerException;
 import org.wso2.carbon.identity.organization.management.service.internal.OrganizationManagementDataHolder;
+import org.wso2.carbon.user.api.Tenant;
```
Comment (dewniMW) [inline components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java:38]: why is this required here?

**Review by thanujalk — APPROVED** [review]
